packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -424,27 +431,47 @@ class CupertinoFullscreenDialogTransition extends StatelessWidget {
   /// Creates an iOS-style transition used for summoning fullscreen dialogs.
   CupertinoFullscreenDialogTransition({
     Key key,
-    @required Animation<double> animation,
```

Comment:
we're renaming an existing public API but I'm surprised this didn't break any tests 😅
given our latest breaking change policy, it means this one's good to go https://flutter.dev/docs/resources/compatibility

Reply:
Yeah this part of flutter isnt very well tested to be honest